src/libasr/codegen/asr_to_c.cpp

```diff
@@ -516,6 +516,14 @@ R"(
         src = out;
     }
 
+    void visit_ComplexConstant(const ASR::ComplexConstant_t &x) {
+        std::string re = std::to_string(x.m_re);
+        std::string im = std::to_string(x.m_im);
+        headers.insert("complex");
+        src = "CMPLX(" + re + ", " + im + ")";
+        last_expr_precedence = 2;
+    }
```

Comment:
I think this same feature is being implemented in #1108 ?

Reply:
Yeah maybe because it was giving errors in multiple files while generating C code.